vllm/v1/core/kv_cache_utils.py

```diff
@@ -646,3 +676,12 @@ def unify_kv_cache_configs(kv_cache_configs: list[KVCacheConfig]):
         kv_cache_config.num_blocks = min_num_blocks
 
     return kv_cache_configs
+
+
+@dataclass
+class PrefixLengthRange:
```

Comment:
Can we use named tuple instead? Instantiating a class object is quite expensive in Python.